src/main/java/com/mineinabyss/looty/ecs/systems/ConsumeOnDurabilityDepleteSystem.kt

```diff
@@ -0,0 +1,17 @@
+package com.mineinabyss.looty.ecs.systems
+
+import com.mineinabyss.geary.ecs.api.entities.GearyEntity
+import com.mineinabyss.looty.ecs.components.ConsumeOnDurabilityDepleteComponent
+import com.mineinabyss.looty.ecs.components.DurabilityDepletedComponent
+import com.mineinabyss.looty.ecs.components.PlayerInventoryContext
+
+object ConsumeOnDurabilityDepleteSystem : LootyItemSystem() {
+    private val durabilityDepletedComponent = has<DurabilityDepletedComponent>()
+    private val context by get<PlayerInventoryContext>()
+
+    override fun GearyEntity.tick() {
+        if (!has<ConsumeOnDurabilityDepleteComponent>()) return
```

Comment:
@0ffz, may I ask you to take a look? 
When I use 
`private val consumeOnDurabilityDepleteComponent = has<ConsumeOnDurabilityDepleteComponent>()`
instead, this system doesn't able to register any archetype.

I wasted so much time at debug but still can't get why it unables to match any archetype.
I use such looty config for testing 
[meteor2.txt](https://github.com/MineInAbyss/Looty/files/6658421/meteor2.txt)